### ISSUE TITLE
Block Editor: Unable to Edit Text with Screen Reader

### ISSUE THREAD

**Issue by donalirl** [wordpress-mobile/WordPress-Android#11765]
### Expected behavior
I expect to be able to edit the text on posts and pages while using the app with a screen reader.

### Actual behavior
I cannot edit the text using the app with a screen reader.

Screen recording: https://d.pr/v/gO0VqP

### Steps to reproduce the behavior
1. Open and page or post in the app with paragraph blocks
2. Using only a screen reader, attempt to edit text.
